osu.Game.Rulesets.Mania/UI/DrawableManiaJudgement.cs

```diff
@@ -32,8 +36,20 @@ public override void PlayAnimation()
                 switch (Result)
                 {
                     case HitResult.None:
+                        this.FadeOutFromOne(800);
+                        break;
+
                     case HitResult.Miss:
-                        base.PlayAnimation();
+                        this.ScaleTo(1.6f);
```

Comment:
Can you explain the relevance of this code added here?

Reply:
Because I've changed the position of the triangles judgement, I have to inline the code in `base.PlayAnimation` and specify the Y position to be `judgement_y_position` instead of zero like in base.